### ISSUE TITLE
trace 실행 결과 채점 관련 질문드립니다.

### ISSUE THREAD

**Issue by YueunLee** [SNU-ARC/2024_spring_sysprog_Lab4#20]
안녕하세요.

trace 파일 실행 결과가 환경에 따라 달라질 수 있는지 궁금합니다.
정확히는 제공된 VM 환경에서 reference와 제 구현이 trace 파일들에 대해 동일한 결과를 낸다면,
채점 환경 상에서도 마찬가지로 동일할 것이라 생각해도 되는지 알고 싶습니다.

감사합니다.

**Comment by yunjayh** [comment]
네. 그렇게 생각하시면 됩니다. 단, VM에서 충분한 반복 시행으로 corner case가 없는지 확인해보시면 좋을 것 같습니다.
